components/feature/awesomebar/src/main/java/mozilla/components/feature/awesomebar/provider/HistoryStorageSuggestionProvider.kt

```diff
@@ -15,29 +15,29 @@ import mozilla.components.feature.awesomebar.facts.emitHistorySuggestionClickedF
 import mozilla.components.feature.session.SessionUseCases
 import java.util.UUID
 
-internal const val HISTORY_SUGGESTION_LIMIT = 20
+internal const val DEFAULT_HISTORY_SUGGESTION_LIMIT = 20
 
 /**
  * A [AwesomeBar.SuggestionProvider] implementation that provides suggestions based on the browsing
  * history stored in the [HistoryStorage].
  *
- * @property historyStorage and instance of the [HistoryStorage] used
+ * @param historyStorage and instance of the [HistoryStorage] used
  * to query matching history records.
- * @property loadUrlUseCase the use case invoked to load the url when the
+ * @param loadUrlUseCase the use case invoked to load the url when the
  * user clicks on the suggestion.
- * @property icons optional instance of [BrowserIcons] to load fav icons
+ * @param icons optional instance of [BrowserIcons] to load fav icons
  * for history URLs.
  * @param engine optional [Engine] instance to call [Engine.speculativeConnect] for the
  * highest scored suggestion URL.
- * @param maxNumberOfResults optional parameter allowing to lower the number of returned suggested
- * history items to below the default of 20
+ * @param maxNumberOfSuggestions optional parameter allowing to specify a different number than the
+ * default of 20 for how many history results to be returned.
```

Comment:
nit: optional parameter to specify the maximum number of returned suggestions, defaults to [DEFAULT_HISTORY_SUGGESTION_LIMIT].